14_functions_lab/01.py

```diff
@@ -1,7 +1,19 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+'''
+mysum calcs sum of parameters while ignoring non-int items
+mymul calcs mul of parameters while ignoring non-int items
+'''
+
+
+def mysum(*n):
+    return sum(i for i in n if type(i) is int)

```

Comment:
numbers.Number is a better indicator for numeric value, since it also covers floats. Here's how to use it:

```
import numbers
def mysum(*n):
  return sum(i for i in n if isinstance(i, numbers.Number))
```